spk/python311-wheels/Makefile

```diff
@@ -0,0 +1,155 @@
+SPK_NAME = python311-wheels
+SPK_VERS = 1.0
+SPK_VERS_MAJOR_MINOR = $(word 1,$(subst ., ,$(SPK_VERS))).$(word 2,$(subst ., ,$(SPK_VERS)))
+SPK_REV = 1
+SPK_ICON = src/python3-pip.png
+
+# Platform "powerpc-none-linux-gnuspe" with compiler "gcc" is not supported by the
+# CPython core team, see https://peps.python.org/pep-0011/ for more information.
+# And compiler must support std=c++11 (OlD_PPC_ARCHS fail, but ARMv5_ARCHS have no issue).
+UNSUPPORTED_ARCHS = $(OLD_PPC_ARCHS)
+
+PYTHON_PACKAGE = python311
+
+MAINTAINER = SynoCommunity
+DESCRIPTION =  Collection of programs for the Python programming language.
+DESCRIPTION_FRE = Collection of programs for the Python programming language.
+DESCRIPTION_SPN = Colección de programas para el lenguaje de programación Python.
+STARTABLE = no
+DISPLAY_NAME = Python Wheels Collection
```

Comment:
Since we now have python311-wheels and python312-wheels we must have different display names.
Even we are not going to publish those packages, we should have different names.
Otherwise it is hard to distinguish the packages in the Package Center if you have installed both of them.

Reply:
And if we do add all/most wheels in those test packages, we could have an installer like regular py packages to simulate apps install as well.  We could then test next changes more easily potentially.

Reply:
And if we do add all/most wheels in those test packages, we could have an installer like regular py packages to simulate apps install as well.  We could then test next changes more easily potentially.